simple_pdf_generator/Cargo.toml

```diff
@@ -13,9 +13,7 @@ repository = "https://github.com/Massimiliano-solutiontech/simple_pdf_generator"
 edition = "2021"
 
 [dependencies]
-chromiumoxide = { version = "0.5.1", features = [
-    "tokio-runtime",
-], default-features = false }
+chromiumoxide = { version = "0.5.1", features = ["tokio-runtime", "fetcher"], default-features = false }
```

Comment:
The `fetcher` feature should be added only when is stable on the `chromiumoxide` crate, maybe you can add as an optional feature as well in this crate

Reply:
Removed it!